styleci.yml

```diff
@@ -0,0 +1,6 @@
+preset: recommended
+risky: true
+finder:
+  exclude:
+    - vendor
+  name: "*.php"
```

Comment:
Is this just for plug-and-play to make StyleCI work? (I haven't used StyleCI before)

Reply:
I use a free opensource plan https://styleci.io to find and fix code style errors. I hope you like it too :) 
It is not supported forks, but I auto-fix code style by IDE